spec.html

```diff
@@ -36833,7 +36833,7 @@ <h1>get ArrayBuffer.prototype.byteLength</h1>
           1. Let _O_ be the *this* value.
           1. Perform ? RequireInternalSlot(_O_, [[ArrayBufferData]]).
           1. If IsSharedArrayBuffer(_O_) is *true*, throw a *TypeError* exception.
-          1. If IsDetachedBuffer(_O_) is *true*, throw a *TypeError* exception.
+          1. If IsDetachedBuffer(_O_) is *true*, return 0.
```

Comment:
this isn't a math value, it's a Number; not sure if this is fine now and can wait for #2007, or if this should be:
```suggestion
          1. If IsDetachedBuffer(_O_) is *true*, return *+0*.
```

Reply:
We'll probably have #2007 merged before this reaches consensus, so I'd say it's best to match the conventions from there. But feel free to wait until it lands.

Reply:
Note though, that #2007 doesn't currently change the existing cases of `If IsDetachedBuffer(_buffer_) is *true*, return 0.` in %TypedArray%.prototype.{byteLength, byteOffset}.

Reply:
Yeah, we will be doing a lot of work on it this week.

Reply:
Sounds good, I've gone ahead and addressed this then.